packages/ui/BudgetIcon/index.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+import { ReactNode } from "react";
+import { Icon, IconType } from "../Icon";
+
+type BudgetIconProps = {
+  icon?: IconType;
+  children: ReactNode;
+};
+
+const BudgetIconStyled = styled.div`
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  height: 100%;
+  width: 100%;
+  background-color: ${({ theme }) => theme.budgetIcon.background};
+  color: ${({ theme }) => theme.budgetIcon.main};
+`;
+
+export const BudgetIcon = ({ icon, children }: BudgetIconProps) => {
+  return (
+    <BudgetIconStyled>
+      {icon ? (
+        <>
+          <Icon icon={icon} iconSize={40} />
```

Comment:
```suggestion
          <Icon icon={icon} />
```
We don't want this if we want to get responsive icon

Reply:
As discussed on the call, icon component will now be extended and it will set font size in em units.